test/index.js

```diff
@@ -34,8 +34,9 @@ test('uses external CJS helpers', snapshot, 'esm-helpers', {modules: 'commonjs',
 test('cherry picks lodash modules', snapshot, 'lodash')
 test('does not cherry pick lodash modules', snapshot, 'lodash', {lodash: false})
 
-test('wraps prop types', snapshot, 'prop-types')
-test('wraps prop types', snapshot, 'prop-types', {removePropTypes: 'wrap'})
+test('wraps prop types assignment', snapshot, 'prop-types')
+test('wraps prop types assignment', snapshot, 'prop-types', {removePropTypes: 'unsafe-wrap'})
+test('wraps prop types value', snapshot, 'prop-types', {removePropTypes: 'wrap'})
```

Comment:
What's the reasoning behind leaving in this test?

Reply:
Just in case a consumer wants to manually use that option for a specific reason (maybe orama, to be extra careful?)

Reply:
Cool, it just felt like it's slightly more of a test for the ability to override a value in the config.

Reply:
Yeah it is, but the whole suite really just tests that certain config/env values result in expected transpilation output.